### ISSUE TITLE
Improper computation of `centroid` in `measure._moments`

### ISSUE THREAD

**Issue by kne42** [scikit-image/scikit-image#2868]
I noticed that the test suite `test_moments.py` did not properly cover cases where `center=None` (its default) in `moments_central`. As the adjusted test failed, I decided to create a test for `centroid` (called to compute the center of the image when none was provided) itself.

It failed:
```python
======================================== FAILURES =========================================
______________________________________ test_centroid ______________________________________

    def test_centroid():
        image = np.zeros((20, 20), dtype=np.double)
        image[14, 14] = 1
        image[15, 15] = 1
        image[14, 15] = 0.5
        image[15, 14] = 0.5
        im_centroid = centroid(image)
>       assert_equal(im_centroid, (14.5, 14.5))
E       AssertionError: 
E       Arrays are not equal
E       
E       (mismatch 100.0%)
E        x: array([ 43.5,  43.5])
E        y: array([ 14.5,  14.5])

skimage/measure/tests/test_moments.py:141: AssertionError
```

This bug was introduced in https://github.com/scikit-image/scikit-image/commit/a346268a461ef2a403ed8e8da6e9e446c741e75b.

**Comment by soupault** [comment]
I think https://github.com/scikit-image/scikit-image/blob/master/skimage/measure/_moments.py#L238 should be normalized by `M[*((0, ) * image.ndim)]`.
/cc @jni

**Comment by kne42** [comment]
@soupault I don't think that's what he meant by `(weighted)`. If this is the case, then this issue and #2869 are irrelevant and should be closed.

My interpretation of that was that each point/pixel in the image was assigned its own weight, i.e. `image[0, 42, 12] = 0.5` would mean that the point `(0, 42, 12)` has a weight of `0.5`. In this situation, you are not normalizing the weighted points individually (that would be counting every point as having an effective weight of `1`), you are normalizing their _sum_ with respect to the weighted sum of all points, which results in the centroid still being relevant to the coordinate system that we started with.

**Comment by jni** [comment]
@kne42's fix in #2869 is correct.
